backend/src/components/rfiApplication.js

```diff
@@ -1,22 +1,42 @@
-'use strict';
-const { sleep, getOperation, postOperation, patchOperationWithObjectId, deleteOperationWithObjectId, minify} = require('./utils');
-const HttpStatus = require('http-status-codes');
-const log = require('./logger');
-const { MappableObjectForFront, MappableObjectForBack } = require('../util/mapping/MappableObject');
-const { RFIApplicationMappings, ServiceExpansionDetailsMappings, DCSWageIncreaseMappings,
-  ExpenseInformationMappings, OtherFundingProgramMappings, IndigenousExpenseMappings} = require('../util/mapping/Mappings');
+"use strict";
```

Comment:
I've added a .prettierrc.json file to make the prettier formatting more consistent with existing code. Try saving this file again.

Reply:
done!!